app/(routes)/pubhub/v1/user/loans/(lib)/requests.ts

```diff
@@ -1,16 +1,81 @@
 import { TUserInfo } from "@/app/(routes)/pubhub/(lib)/types"
 import { getPublizonServiceParameters } from "@/lib/helpers/publizon"
+import { createClientAsync as friendlyCardNumberClientAsync } from "@/lib/soap/publizon/v2_7/generated/getfriendlycardnumber"
 import { createClientAsync } from "@/lib/soap/publizon/v2_7/generated/getlibraryuserorderlist"
 
-import { TLibraryUserOrderList } from "./types"
+import { TFriendlyCardNumberResultSchema, TLibraryUserOrderList } from "./types"
+
+// This is a default response for the library user order list response.
+// In case no data is returned from the getlibraryuserorderlist SOAP service,
+// we will use this default response to avoid errors in the frontend.
+const defaultData = {
+  response: {
+    status: {
+      LibraryExtension: {
+        maxloanpertime: "0",
+        maxloanpertimesound: "0",
+        userebookloansremain: "0",
+        usertotalloans: "0",
+        usertotalebookloans: "0",
+        usertotalsoundloans: "0",
+      },
+    },
+    data: {
+      orderitem: [],
+      friendlycardnumber: "",
+      loans: [],
+      libraryData: {
+        maxAmountPerMonth: 0,
+        maxConcurrentAudiobookLoansPerBorrower: 0,
+        maxConcurrentEbookLoansPerBorrower: 0,
+      },
+      userData: {
+        totalLoans: 0,
+        totalEbookLoans: 0,
+        totalAudioLoans: 0,
+        ebookLoansRemaining: 0,
+        audiobookLoansRemaining: 0,
+        friendlyCardNumber: "",
+      },
+    },
+  },
+} as TLibraryUserOrderList
 
 export const getLibraryUserOrderListRequest = async (uniLoginUserInfo: TUserInfo) => {
   const client = await createClientAsync(
     "./lib/soap/publizon/v2_7/wsdl/getlibraryuserorderlist.wsdl"
   )
-  const [response] = await client.GetLibraryUserOrderListAsync({
+  const [libraryUserResponse] = await client.GetLibraryUserOrderListAsync({
     ...getPublizonServiceParameters(),
     cardnumber: uniLoginUserInfo.uniid,
   })
-  return response.GetLibraryUserOrderListResult as TLibraryUserOrderList
+
+  let libraryUserOrderList =
+    libraryUserResponse.GetLibraryUserOrderListResult as TLibraryUserOrderList
```

Comment:
```suggestion
  const libraryUserOrderList =
    libraryUserResponse.GetLibraryUserOrderListResult as TLibraryUserOrderList
```
becasue of later suggestion (not tested)...